rest/db-file-get.rst

```diff
@@ -1,12 +1,31 @@
 GET /rest/db/file
 =================
 
-Returns most data available about a given file, including version and
-availability. Takes ``folder`` and ``file`` parameters. ``local`` and
-``global`` refer to the current file on disk and the globally newest file,
-respectively.
+Returns data about a given file, including version and availability.
 
-.. code-block::
+Request
+-------
+
+The call requires parameters for `folder` and `file` in the query string:
+
+- `folder` is the folder ID which you can find in the Syncthing Web GUI,
+  e.g. `5camp-slpa8`.
+
+- `file` is the relative path of the file starting from the folder root to the
+  file you are interested in.  The path and filename must be correctly
+  URL-encoded.
+
+Example Request
+^^^^^^^^^^^^^^^
+
+.. code-block:: bash
+
+    curl --silent --get --header "X-API-KEY: YOUR_API_TOKEN" "http://localhost:8384/rest/db/file?folder=YOUR_FOLDER_ID" --data-urlencode "file=path/to/file.pdf"
```

Comment:
There are already numerous examples involving `curl` in the docs.  I'd find it convenient to adopt the same CLI options used there, e.g. `-H` instead of `--header` and omit the `--get`.